build-aux/.run-format.zsh

```diff
@@ -54,33 +54,87 @@ invoke_formatter() {
         exit 2
       fi
 
-      if (( ! #source_files )) source_files=((libobs|libobs-*|UI|plugins|deps|shared)/**/*.(c|cpp|h|hpp|m|mm)(.N))
+      if (( ! #source_files )) source_files=((libobs|libobs-*|UI|plugins|deps)/**/*.(c|cpp|h|hpp|m|mm)(.N))
```

Comment:
Are we intentionally excluding the shared directory?

Reply:
Should just be an automatic remove caused by the rebase.